tardis/adapters/sites/moab.py

```diff
@@ -47,10 +50,29 @@ async def moab_status_updater(executor):
     return moab_resource_status
 
 
+class MoabAdapterConfigurationModel(SiteAdapterBaseModel):
+    """
+    pydantic model for the input validation of the Moab site adapter configuration
+    """
+
+    executor: Optional[Executor] = ShellExecutor()
+    StatusUpdate: PositiveInt
+    StartupCommand: Optional[str] = None
+
+    @root_validator(allow_reuse=True)
+    def deprecate_startup_command(
+        cls, valuesDict: [str, Any]  # noqa B902
+    ) -> Dict[str, Any]:
+        if valuesDict["StartupCommand"] is None:
+            del valuesDict["StartupCommand"]
```

Comment:
It exists and is `None`, because of https://github.com/MatterMiners/tardis/blob/eca339924489767aef329e2151213f674981159c/tardis/adapters/sites/moab.py#L60. However, I think there is no need to set it to `None` before. Let me check.

Reply:
Setting to `StartupCommand = None` is not necessary, it will be automatically set to `None`. I have also simplified the check. Thanks for the hint!